developer-docs-site/docs/guides/move-guides/book/standard-library.md

```diff
@@ -1,542 +1,3 @@
-
-```move
-    public fun requires_address(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Should be used if a role did not match a required role when using RBAC. Constructs an abort code with specified `reason` and category `REQUIRES_ROLE`. Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun requires_role(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Should be used if an account did not have a required capability when using RBAC. Constructs an abort code with specified `reason` and category `REQUIRES_CAPABILITY`. Should be Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun requires_capability(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Should be used if a resource did not exist where one was expected. Constructs an abort code with specified `reason` and category `NOT_PUBLISHED`. Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun not_published(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Should be used if a resource already existed where one was about to be published. Constructs an abort code with specified `reason` and category `ALREADY_PUBLISHED`. Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun already_published(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Should be used if an invalid argument was passed to a function/operation. Constructs an abort code with specified `reason` and category `INVALID_ARGUMENT`. Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun invalid_argument(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Should be used if a limit on a specific value is reached, e.g., subtracting 1 from a value of 0. Constructs an abort code with specified `reason` and category `LIMIT_EXCEEDED`. Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun limit_exceeded(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Should be used if an internal error or bug was encountered. Constructs an abort code with specified `reason` and category `INTERNAL`. Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun internal(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-Used for extension points, should be not used under most circumstances. Constructs an abort code with specified `reason` and category `CUSTOM`. Will abort if `reason` does not fit in 56 bits.
-
-```move
-    public fun custom(reason: u64): u64;
-```
-
----------------------------------------------------------------------------
-
-## fixed_point32
```

Comment:
This can be kept too.

cc @gerben-stavenga for adding docs for the new fixed_point64 later